unet_prediction.py

```diff
@@ -64,15 +86,17 @@ def unet_initialize(input_shape):
 
         # Download the weights
         logger.debug("Downloading model weights to: {:s}".format(weights_filename))
-        id = "146sae_bv2rJa4YoJr8Hlo-MFKfuepoMX"
+        #id = "146sae_bv2rJa4YoJr8Hlo-MFKfuepoMX"
+        id = "1I9j4oABbcV8EnvO_ufACXP9e4KyfHMtE"
```

Comment:
Two things: 

1. I don't think the other model needs to be commented out here, I think it can just be removed (it should still exist in the git history of the file in case we want to go back to it for whatever reason)
1. The variable should be named `model_id` or something that's not `id`, since `id` is a python keyword that's being overwritten here.

Reply:
Great, thanks for the review! I removed the old model and renamed ```id```!